src/cml.js

```diff
@@ -35,12 +39,13 @@ const get_driver = (opts) => {
 
   if (driver === 'github') return new Github({ repo, token });
   if (driver === 'gitlab') return new Gitlab({ repo, token });
+  if (driver === 'bitbucket') return new BitBucketCloud({ repo, token });
 
   throw new Error(`driver ${driver} unknown!`);
 };
 
 const infer_token = () => {
-  const { repo_token, GITHUB_TOKEN, GITLAB_TOKEN } = process.env;
+  const { repo_token, GITHUB_TOKEN, GITLAB_TOKEN} = process.env;
```

Comment:
lets add BITBUCKET_TOKEN to maintain the structure

Reply:
done!